.gitignore

```diff
@@ -1 +1,3 @@
 node_modules
+.idea
```

Comment:
Should be in a global .gitignore, not in this repo-specific one

Reply:
Fixed in latest commit